cdap-ui/templates/cdap-etl-realtime/Stream.json

```diff
@@ -22,8 +22,8 @@
         },
         "name": {
           "widget": "stream-selector",
-          "label" : "Name",
-          "description" : "Name of the stream to which we write to. If the stream is present, then data will be written to it. If it's not present then one would be created.",
+          "label" : "Stream Name",
```

Comment:
I think so. There are three fields there, all ending in "Name". For some reason, the "Stream Name" is last. It is also consistent with others.